NO_AFOX_001 (#359)
NINAnor/ecRxiv: R/NO_AFOX_001.qmd

    "Indicator ID",
    "Indicator Name",
    "Continent",
    "Country",
    "Ecosystem Condition Typology Class", 
    "Realm",
    "Biome",
    "Ecosystem",
    "Year added", 
    "Last update",
    "Version",
    "Version comment",
    "Normalised",
    "Spatial aggregation pathway")) |>   
  kable()

```

:::

::: {.callout-tip collapse="true"}

## Log

<!--# Update this log with short messages for each update -->
- May 27 2026 - submission
:::

<hr />

<!--# Document you work below.  -->

```{r fig-img}
#| fig-cap: 'Arctic fox (photo: Craig Jackson)'
#| echo: false
knitr::include_graphics(here::here("img/Fjellrev_CraigJ.jpg"))

```

## 1. Summary

<!--# 
With a maximum of 300 words, describe the indicator in general terms as you would to a non-expert. Think of this as a kind of common language summary. Follow the four numbers sub-headers below. under point 2, it is possible under  include a bullet point list of the specific steps in the workflow. 
 -->

1. **Background and rationale**:
<!-- What does the metric represent? Why is this relevant for describing ecosystem condition in this ecosystem?
What are the main anthropogenic impact factors? -->

> Arctix fox is an alpine specialist. The populations today are small and fragmented due to past overhunting. Arctic for is sensitive to climate change, perhaps mostly indirectly so, as wetter and warmer winters are causing a decline in rodent numbers, it's main food source. It is also negatively affected by competition with the red fox, and by the effects of having small and isolated population, and a range of anthopogenic disturbance factors. There is a national breeding program for arctic fox, which is the main reason why we see species recovery in some areas.      

Reviewer: ```suggestion
Arctix fox is an alpine specialist. The populations today are small and fragmented due to past overhunting. Arctic fox is sensitive to climate change, perhaps mostly indirectly so, as wetter and warmer winters are causing a decline in rodent numbers, it's main food source. It is also negatively affected by competition with the red fox, and by the effects of having small and isolated population, and a range of anthopogenic disturbance factors. There is a national breeding program for arctic fox, which is the main reason why we see species recovery in some areas.      
```

2. **Methods**:
<!-- What kind of data is used? How is the data customized (modified, estimated, integrated) to fit its purpose as an indicator? -->

> The arctic fox in Norway has it's own monitoring program. The data is stored at [rovbase](https://www.rovbase.no/). This program uses DNA sampling and mark-recapture, among other things [@jackson2025fjellrev].

Reviewer: ```suggestion
The arctic fox in Norway has it's own monitoring program. The data is stored at [rovbase](https://www.rovbase.no/). This program uses DNA sampling and mark-recapture, among other methods [@jackson2025fjellrev].
```

3. **Interpretation**:
<!-- Add a very short description of how the indicator values should be interpreted. Here's an example for a bumblebee indicator: An indicator value of 1 signifies an intact bumblebee community. Values decreasing toward zero reflect negative ecological change, specifically the decline of species expected to be present. This metric does not account for potentially positive changes, such as increases in species' abundance or the arrival of new species. Tip: Add text on the line directly following the subheader, and don't introduce blank lines. This way the whole paragraph renders as a block quote. -->
An indicator value of 1 means that arctic fox is as numerous, or more, than what we expect under the reference condition.
Values decreasing toward zero indicates that the average population size is decreasing.

4. **Results and development needs**:
<!-- Briefly, what does the indicator say about the ecosystem condition. What is the current status of  the metric (can it be used or is it still in development), and what immediate tasks remain to improve the indicator or to make the indicator operational -->

> The indicator value for Norway as a whole in 2024 is 0.05 [95% CI = 0.03-0.06]. This is very low and reflect the near extiction of this species from Norwegian alpine ecosystems. There has been some recovery in some local areas due to active intervention (captive breeding).      

Reviewer: ```suggestion
The indicator value for Norway as a whole in 2024 is 0.05 [95% CI = 0.03-0.06]. This is very low and reflects the near extiction of this species from Norwegian alpine ecosystems. There has been some recovery in some local areas due to active intervention (captive breeding).      
```

2. **Metoder**: 
Fjellrev har sitt eget overvåkingsprogram. Dataene lagres i [rovbase](https://www.rovbase.no/). Programmet benytter blant annet DNA-prøver og fangst-gjennfangst for å estimere bestandsstørrelsen [@jackson2025fjellrev].
Dataene hentes fra databasen for Naturindeks for Norge [@nater_naturindeks_2025].

3. **Tolkning**: 
En indikatorverdi på 1 betyr at fjellrevbestanden er like stor eller større enn det vi forventer under referansetilstanden. Verdier som nærmer seg null indikerer at den gjennomsnittlige bestandsstørrelsen på fylkesnivå er avtagende.

4. **Resultater og utviklingsbehov**: 
Indikatorverdien for hele Norge i 2024 er 0,05 [95 % KI = 0,03–0,06]. Dette er svært lavt og gjenspeiler at arten har vært, og delvis fortsatt er nær utryddelse fra norske alpine økosystemer. Det har vært noe bestandsøkning i enkelte lokale områder som følge av aktive tiltak, særlig gjennom foring, oppdrett og utsetting.

## 2. About the underlying data

<!--# Give a general introduction the datasets you have used, and provide more specific details in the sub-chapters.  -->

> We take data from the Norwegian Nature index [@nater_naturindeks_2025]. Originally, the data from the arctic fox monitoring is stored at www.rovbase.no, and comes from the national arctic fox minitoring program [@jackson2025fjellrev]. 

Reviewer: ```suggestion
We take data from the Norwegian Nature index [@nater_naturindeks_2025]. Originally, the data from the arctic fox monitoring is stored at www.rovbase.no, and comes from the national arctic fox monitoring program [@jackson2025fjellrev]. 
```

### 2.1 Spatial and temporal resolution and extent

<!--# Describe the temporal and spatial resolution and extent of the data used -->

> Indicator values are available for 1990, 2000, and every yer from 2008 to 2024. Each years estimate has been averaged out over three years as a moving window. 

Reviewer: ```suggestion
Indicator values are available for 1990, 2000, and every year from 2008 to 2024. Each year's estimate has been averaged out over three years as a moving window. 
```

B1 - Compositional state characteristics

<!-- 
### 3.1.1 Other condition typologies -->

<!--# Optional: Add text about other standards or typologies, such as alternative national condition typologies, and how the indicator relates to these -->

### 3.2 Ecosystem condition characteristic

<!--# 

Describe the ecosystem condition characteristic represented in the indicator. See 10.3897/oneeco.6.e58218 for information on what these characteristics might be.
For example, and indicator called 'Trenching in mires' could be made to represent an ecosystem characteristic 'Intact hydrology'. The term 'characteristic' is used similar to the term 'criteria' in Multiple Criteria Decision Making.  

-->

The ecosystem characteristic represented in this indicator is an ecosystem where species abundances and composition is similar to that in the reference condition. 

### 3.3 Collinearities with other indicators

<!--# Describe known collinearities with other metrics (indicators or variables) that could become problematic if they were also included in the same Ecosystem Condition Assessment as the indicator described here. -->

> The arctic fox indicator is assumed to be corralted to the small rodent indicator (NO_ROED_001). 

Reviewer: ```suggestion
The arctic fox indicator is assumed to be correlated to the small rodent indicator (NO_ROED_001). 
```

### 3.4 Impact factors

<!--# Describe the main natural and anthropogenic factors that affecst the metric -->

> Arctic fox is negatively affected by climate change (esp. through its effect in small rodents). Habitat degradation and land use change also has a negative impact. 

Reviewer: ```suggestion
Arctic fox is negatively affected by climate change (esp. through its effect on small rodents). Habitat degradation and land use change also has a negative impact. 
```

### 3.5 Spatial aggregation

<!-- Describe in word the same aggregation pathway presented in the top yaml (the metadata). If you have scaled against the reference value, it is especially important to know if you did any avergaing or summations etc. beforehand. An example text for a rather simple case could be: "The variable is scaled at the initial polygon level and the point estimate is a area-weighted arithmetic mean of all polygins. There is no transformation, and truncation is not needed as the indicator becomes naturally bound between 0 and 1".  -->
The data is available as the scale of municiaplities. At this point I normalise (scale and truncate) the variable. I then aggregate these, first toe the five geographic regions, and then from there, to a national estimate.

## 4. Reference condition and levels

### 4.1 Reference condition

<!--# Define the reference condition (or refer to where it is defined). Note the distinction between reference condition and reference levels 10.3897/oneeco.5.e58216  -->

> The reference condition is a moutain ecosystem with minimal human impact and a climate similar to the period 1961-1990 [@nybo_fagsystem_2017].

Reviewer: ```suggestion
The reference condition is a mountain ecosystem with minimal human impact and a climate similar to the period 1961-1990 [@nybo_fagsystem_2017].
```

The reported uncertainties reflect the uncertainties around the true parameter means, i.e. they are inferrential uncertainties for the true value of the indicator for the given area and time combination.  

## 6. References

<!--# You can add references manually or use a citation manager and add intext citations as with crossreferencing and hyperlinks. See https://quarto.org/docs/authoring/footnotes-and-citations.html -->

::: {#refs}
:::

## 7. Import and prepare datasets

<!--# All the data import and preparation should be done in this chapter, before the analysis chapter. As a minimum, add code for importing data, and describe or show the data structure. You may also explain certain variables, what they represent, and their distributions -->

### 7.1 Naturindeks

<!--# Import and prepare the first dataset for analysis. Change the header from Dataset A to the name of the actual dataset. -->

We download the dataset from the Naturindeks database using the API. 

> The function giver read-only access without requiring a username and password.

Reviewer: ```suggestion
The function gives read-only access without requiring a username and password.
```
